src/zenml/cli/login.py

```diff
@@ -1042,7 +1042,7 @@ def logout(
                 credentials_store.clear_credentials(server_url=server)
             cli_utils.declare(
                 "Logged out from ZenML Pro.\n"
-                f"Hint: You can run 'zenml login {credentials.server_name}' to "
+                f"Hint: You can run 'zenml login \"{credentials.server_name}\"' to "
```

Comment:
This should be single quotes (`'`) instead of double quotes (`"`) to account for tenant names with a `$` sign in them. Same for the other message below.